client/components/Voting/voting.tsx

```diff
@@ -108,7 +117,39 @@ export default class VotingForms extends React.Component<
         : this.state.finalvoteData,
       didFinalvote,
       votingFinalized,
+      prevotedPartners: company.partnerVotes.prevote,
+      finalvotedPartners: company.partnerVotes.final,
     });
+
+    // Setup autorefresh every 3 seconds to pull updated company information
+    this.interval = setInterval( async() => {
+      const updatedCompany = await client
+        .service('api/companies')
+        .get(this.props.companyID);
+
+      /*
+       * Determine if we need to update our didPrevote or didFinalVoteStew
+       */
+      let newDidPrevote = this.state.didPrevote;
+      let newDidFinalvote = this.state.didFinalvote;
+      updatedCompany.partnerVotes.prevote.forEach((partnerObj: PartnerVoteObj) => {
+        newDidPrevote = newDidPrevote || (partnerObj.partner_id == this.props.user.id);
+      });
+
+      updatedCompany.partnerVotes.final.forEach((partnerObj: PartnerVoteObj) => {
+        newDidFinalvote = newDidFinalvote || (partnerObj.partner_id == this.props.user.id);
+      });
+
+      this.setState({prevotedPartners: updatedCompany.partnerVotes.prevote,
```

Comment:
It’s probably better to make a new timeout every time, that’ll kill a lot of redraw and timing errors.

Reply:
Sorry not quite sure I understand.